src/map/script.c

```diff
@@ -13841,14 +13841,23 @@ BUILDIN(undisguise)
  * @type unused
  *------------------------------------------*/
 BUILDIN(classchange) {
-	int class, type;
+	int class, type, target;
 	struct block_list *bl=map->id2bl(st->oid);
-
+	
 	if(bl==NULL) return true;
```

Comment:
since the command is changed heavily, please adjust this to current coding style, too
`if (bl == NULL)`